sdk/rms_sdk/RestClients/LicenseParser.cpp

```diff
@@ -39,7 +40,14 @@ const uint8_t BOM_UTF8[] = {0xef, 0xbb, 0xbf};
 const shared_ptr<LicenseParserResult> LicenseParser::ParsePublishingLicense(const void *pbPublishLicense,
                                                                 size_t cbPublishLicense)
 {
+  if(!QCoreApplication::instance()) {
+    int argc = 1;
+    char name[] = "LicenseParser::ParsePublishingLicense";
+    char* argv = &name[0];
+    QCoreApplication a(argc, &argv);
     return ParsePublishingLicenseInner(pbPublishLicense, cbPublishLicense);
+  }
+  return ParsePublishingLicenseInner(pbPublishLicense, cbPublishLicense);
```

Comment:
Do you need return both inside and outside the if statement? 
Can you just return once outside here and other places?

Reply:
Yes and No, respectively. Essentially, an instance of QCoreApplication should exist in the scope from which the subroutine that needs it gets called.

Reply:
I see. Thought so just wanted to confirm